internal/cli/auth/login.go

```diff
@@ -266,6 +268,7 @@ func LoginBuilder() *cobra.Command {
 	cmd.Flags().BoolVar(&opts.IsGov, "gov", false, "Log in to Atlas for Government.")
 	cmd.Flags().BoolVar(&opts.NoBrowser, "noBrowser", false, "Don't try to open a browser session.")
 	cmd.Flags().BoolVar(&opts.SkipConfig, "skipConfig", false, "Skip profile configuration.")
+	_ = cmd.Flags().MarkDeprecated("skipConfig", "if profile is configured, login flow skips by default the config step.")
```

Comment:
Note here, the wording is a little strange. I'd suggest:

'if profile is configured, the login flow will skip the config step by default.'

Reply:
thanks for the suggestion. 
done in this PR https://github.com/mongodb/mongodb-atlas-cli/pull/1218/files